### PR TITLE
auto bump the azure storage cli

### PR DESCRIPTION
**Pull request by fmoehler**
### What is this change about?

We introduce automatic bumping for the azure storage cli

### Please provide contextual information.

As the azure-storage-cli is now being bumped automatically, we also want to do it within bosh

### What tests have you run against this PR?

Because we don't have enough permissions in the cloudfoundry org, we could not test this so far

### How should this change be described in bosh release notes?

Introduce auto bumping for azure-storage-cli

### Does this PR introduce a breaking change?

No

### Tag your pair, your PM, and/or team!
@friegger

### CODE REVIEW

**Review by lnguyen — APPROVED** [review]
LGTM

File: ci/pipeline.yml [hunk 1]
```diff
@@ -1107,6 +1109,22 @@ jobs:
             options:
               credentials_source: static
               json_key: '((bosh_release_blobs_gcp_credentials_json))'
+    - task: bump-bosh-blobstore-azure-storage
+      file: bosh-src/ci/tasks/bump-blobstore-cli.yml
```
Comment (mvach) [inline ci/pipeline.yml:1113]: I noticed that `bosh-src/ci/tasks/bump-blobstore-cli.sh` is checking the blobstore type.

The configured `BLOBSTORE_TYPE: azure-storage` would result in `"Error: unknown BLOBSTORE_TYPE='${BLOBSTORE_TYPE}'; exiting"`

Don't we need to update `bosh-src/ci/tasks/bump-blobstore-cli.sh` first?

**Review by mvach — CHANGES_REQUESTED** [review]
lgtm

**Review by mvach — APPROVED** [review]

**Review by rkoster — APPROVED** [review]

**Comment by rkoster** [comment]
This PR is blocked on further director changes which actually use the azure cli

**Comment by beyhan** [comment]
We have enough reviews and https://github.com/cloudfoundry/bosh/pull/2456 got approved and merged. That is why I will merge this one also.

**Pull request merged**